packages/main/src/webComponents/UploadCollection/index.tsx

```diff
@@ -123,7 +111,7 @@ const UploadCollection = withWebComponent<UploadCollectionPropTypes, UploadColle
   ['accessibleName', 'noDataDescription', 'noDataText', 'selectionMode'],
   ['hideDragOverlay'],
   ['header'],
-  ['drop', 'item-delete', 'selection-change']
```

Comment:
The `drop` event has been removed, but as the native event is still published I guess that should be fine and not a breaking change.